recipes/knem-headers/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "1.1.4" %}
+{% set build = 0 %}
+
+package:
+  name: knem-headers
+  version: {{ version }}
+
+source:
+  url: https://gitlab.inria.fr/knem/knem/-/archive/knem-{{ version }}/knem-knem-{{ version }}.tar.gz
+  sha256: 2196777458626c467057dd7eaee915154acda47dcd0600edb271ef7535198e7b
+
+build:
+  number: {{ build }}
+  skip: true  # [win or osx]
+
+  run_exports:
+    - {{ pin_subpackage('knem-headers', max_pin='x.y.z') }}
```

Comment:
If these are only headers, there should be no runtime dependency.

Reply:
OK